### PR TITLE
Ingesting 2020 EMNLP, Findings, workshops, and CoNLL

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

**Comment by mjpost** [comment]
This is a work in progress; please allow me to merge. However, reviews would be welcome.

There are a few ambiguous names that I resolved, could use checking:

* I think there may be a new Yang Liu (2020.findings-emnlp.132)?
* There is a Bo Li in both EMNLP and Findings that could perhaps be split out

**Review by danielgildea — COMMENTED** [review]
Lots of all upper and all lower.
`  % grep author 2020.emnlp.xml | egrep '>[A-Z]+<' `
`  % grep author 2020.emnlp.xml | egrep '>[a-z]+<'`

**Comment by mjpost** [comment]
Ah. I think I'll just apply initial-casing to them all.

File: data/xml/2020.emnlp.xml [hunk 1]
```diff
@@ -0,0 +1,7608 @@
...
+      <title><fixed-case>BERT</fixed-case>-<fixed-case>ATTACK</fixed-case>: Adversarial Attack against <fixed-case>BERT</fixed-case> Using <fixed-case>BERT</fixed-case></title>
+      <author><first>Linyang</first><last>Li</last></author>
+      <author><first>Ruotian</first><last>Ma</last></author>
+      <author><first>Qipeng</first><last>Guo</last></author>
+      <author><first>Xiangyang</first><last>Xue</last></author>
+      <author><first>Xipeng</first><last>Qiu</last></author>
+      <pages>6193–6202</pages>
+      <url hash="604993f5">2020.emnlp-main.500</url>
+    </paper>
+    <paper i
```
Comment (danielgildea) [inline data/xml/2020.emnlp.xml:7141]: garbled

File: data/xml/2020.emnlp.xml [hunk 2]
```diff
@@ -0,0 +1,7608 @@
...
+      <title><fixed-case>BERT</fixed-case>-<fixed-case>ATTACK</fixed-case>: Adversarial Attack against <fixed-case>BERT</fixed-case> Using <fixed-case>BERT</fixed-case></title>
+      <author><first>Linyang</first><last>Li</last></author>
+      <author><first>Ruotian</first><last>Ma</last></author>
+      <author><first>Qipeng</first><last>Guo</last></author>
+      <author><first>Xiangyang</first><last>Xue</last></author>
+      <author><first>Xipeng</first><last>Qiu</last></author>
+      <pages>6193–6202</pages>
+      <url hash="604993f5">2020.emnlp-main.500</url>
+    </paper>
+
```
Comment (danielgildea) [inline data/xml/2020.emnlp.xml:7143]: editor names garbled

**Review by danielgildea — APPROVED** [review]

**Comment by mjpost** [comment]
Workshops coming next...

**Comment by mjpost** [comment]
@mbollmann I'd be interested in your thoughts on automatic venue-adding, if you have time to look (not a big change, in case you don't have time):

* https://github.com/acl-org/acl-anthology/pull/1045/files#diff-ca1151d331faef3c1e23dc91bff20750afcbaa689768cc3e4746d1177ac85d79
* https://github.com/acl-org/acl-anthology/pull/1045/files#diff-9a14f867033c7ba99b580c799cd4d6563ffdc469096b929c6add50354271ad38

**Review by danielgildea — APPROVED** [review]

**Comment by mjpost** [comment]
I'm not sure how I missed these caps—the ingestion script should have caught them.

**Comment by mjpost** [comment]
@danielgildea I plan to merge today so these can go live—should I wait a bit for more changes?

**Comment by danielgildea** [comment]
> should I wait a bit for more changes?
all set, you can go ahead and merge

**Pull request merged**